hcx-apis/src/main/java/org/swasth/hcx/utils/SearchUtil.java

```diff
@@ -17,13 +17,10 @@
 
 public final class SearchUtil {
 
-    private SearchUtil() {}
+    SearchUtil() {}
```

Comment:
Do not change the access modifier for constructor

Reply:
done